include/deal.II/base/path_search.h

```diff
@@ -75,11 +75,11 @@ DEAL_II_NAMESPACE_OPEN
  * characters are not added automatically (allowing you to do some real file
  * name editing).
  *
- * @todo Add support for environment variables like in kpathsea.
+ * @deprecated Use std::filesystem or boost::filesystem facilities instead.
```

Comment:
We don't ship `boost::filesystem`'s source files, only the headers. Let's just not recommend it.